spec.html

```diff
@@ -47616,21 +47616,22 @@ <h1>%WrapForValidIteratorPrototype%.return ( )</h1>
 
         <emu-clause id="sec-iterator.prototype">
           <h1>Iterator.prototype</h1>
-          <p>The initial value of Iterator.prototype is %Iterator.prototype%.</p>
+          <p>The initial value of Iterator.prototype is the Iterator prototype object.</p>
           <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
         </emu-clause>
       </emu-clause>
     </emu-clause>
 
     <emu-clause oldids="sec-%iteratorprototype%-object" id="sec-%iterator.prototype%-object">
-      <h1>The %Iterator.prototype% Object</h1>
-      <p>The <dfn>%Iterator.prototype%</dfn> object:</p>
+      <h1>The Iterator Prototype Object</h1>
+      <p>The <dfn>Iterator prototype object</dfn>:</p>
       <ul>
         <li>has a [[Prototype]] internal slot whose value is %Object.prototype%.</li>
+        <li>is <dfn>%Iterator.prototype%</dfn>.</li>
         <li>is an ordinary object.</li>
```

Comment:
(optional)
The list in other "Foo Prototype Object" sections mostly uses the following order:
  * is %Foo.prototype%.
  * is an ordinary object
  * has a [[Prototype]] internal slot whose value is ...

So, while you're here, it might be nice to reorder this as well?